### PR TITLE
Various tweaks

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: Pock/AppDelegate.swift [hunk 1]
```diff
@@ -47,10 +47,10 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             button.image?.isTemplate = true
             /// Create menu
             let menu = NSMenu(title: "Menu")
-            menu.addItem(withTitle: "Preferences", action: #selector(openPreferences), keyEquivalent: "")
+            menu.addItem(withTitle: "Preferences…", action: #selector(openPreferences), keyEquivalent: ",")
             // menu.addItem(withTitle: "Customize", action: #selector(openCustomization), keyEquivalent: "")
             menu.addItem(NSMenuItem.separator())
-            menu.addItem(withTitle: "Quit Pock.", action: #selector(NSApp.terminate(_:)), keyEquivalent: "")
+            menu.addItem(withTitle: "Quit Pock", action: #selector(NSApp.terminate), keyEquivalent: "q")
```
Comment (sindresorhus) [inline Pock/AppDelegate.swift]: Even menu bar apps should have keyboard shortcuts.

File: Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift [hunk 2]
```diff
@@ -26,10 +26,10 @@ final class GeneralPreferencePane: NSViewController, Preferenceable {
     
     /// Preferenceable
     let toolbarItemTitle: String   = "General"
-    let toolbarItemIcon:  NSImage  = NSImage(named: NSImage.Name("pock-icon"))!
-    
+	let toolbarItemIcon:  NSImage  = NSImage(named: .preferencesGeneral)!
```
Comment (sindresorhus) [inline Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift]: You should prefer using the system icon for this. It's what users are used to. And it looks weird to have the app icon there.

**Comment by sindresorhus** [comment]
I also don't think you should have the app icon and version in the preference pane. It would be better to have an "About" item in the menu bar menu instead. Just a recommendation. I have a lot of experience with making preference panes.

**Comment by sindresorhus** [comment]
I would also move the "Check for updates" button to the menu bar menu. It's not a preference. Don't overload the preference pane with random things just because you have the space.

**Comment by pigigaldi** [comment]
I'm closing this since Preferences will be redesigned but, of course, I'll take in consideration all these points for the next huge update.
Thank you very much for the inputs! :)

**Pull request closed without merging**